docker/node/entry.sh

```diff
@@ -1,72 +1,72 @@
-#!/bin/bash
+#!/bin/sh
 
-set -Eeuo pipefail
+set -euo pipefail
 
 usage() {
-	echo -e \
-	"Usage:\n" \
-	"  $0 nano_node [daemon] [cli_options] [-l] [-v size]\n" \
-	"    daemon\n" \
-	"      start as daemon\n\n" \
-	"    cli_options\n" \
-	"      nano_node cli options <see nano_node --help>\n\n" \
-	"    -l\n" \
-	"      log to console <use docker logs {container}>\n\n" \
-	"    -v<size>\n" \
-	"      vacuum database if over size GB on startup\n\n" \
-	"  $0 bash [other]\n" \
-	"    other\n" \
-	"      bash pass through\n" \
-	"  $0 [*]\n" \
-	"    *\n" \
-	"      usage\n\n" \
-	"default:\n" \
-	"  $0 nano_node daemon -l"
+	printf "Usage:\n"
+	printf "  $0 nano_node [daemon] [cli_options] [-l] [-v size]\n"
+	printf "    daemon\n"
+	printf "      start as daemon\n\n"
+	printf "    cli_options\n"
+	printf "      nano_node cli options <see nano_node --help>\n\n"
+	printf "    -l\n"
+	printf "      log to console <use docker logs {container}>\n\n"
+	printf "    -v<size>\n"
+	printf "      vacuum database if over size GB on startup\n\n"
+	printf "  $0 bash [other]\n"
+	printf "    other\n"
+	printf "      bash pass through\n"
+	printf "  $0 [*]\n"
+	printf "    *\n"
+	printf "      usage\n\n"
+	printf "default:\n"
+	printf "  $0 nano_node daemon -l\n"
 }
 
 OPTIND=1
-command=()
-IFS=' ' read -r -a TEMP_OPTS <<<"$@"
-passthrough=()
+command=""
+passthrough=""
 db_size=0
 log_to_cerr=0
 
-if [ ${#TEMP_OPTS[@]} -lt 2 ]; then
+if [ $# -lt 2 ]; then
 	usage
 	exit 1
 fi
 
-if [[ "${TEMP_OPTS[0]}" = 'nano_node' ]]; then
-	unset 'TEMP_OPTS[0]'
-	command+=("nano_node")
+if [ "$1" = 'nano_node' ]; then
+	command="${command}nano_node"
 	shift;
-	for i in "${TEMP_OPTS[@]}"; do
+	for i in $@; do
 		case $i in
 			"daemon" )
-				command+=("--daemon")
+				command="${command} --daemon"
 				;;
 			* )
-				passthrough+=("$i")
+				if [ ${#passthrough} -ge 0 ]; then
+					passthrough="${passthrough} $i"
+				else
+					passthrough="$i"
+				fi
 				;;
 		esac
 	done
-	for i in "${passthrough[@]}"; do
-		if [[ "$i" =~ ^"-v" ]]; then
-		        db_size=${i//-v/}
-			echo "Vacuum DB if over $db_size GB on startup"
-		elif [[ "$i" = '-l' ]]; then
-			echo "log_to_cerr = true"
-			command+=("--config")
-			command+=("node.logging.log_to_cerr=true")
-		else
-		 	command+=("$i")
-		fi
+	for i in $passthrough; do
+		case $i in
+			*"-v"*)
+				db_size=$(echo $i | tr -d -c 0-9)
+				echo "Vacuum DB if over $db_size GB on startup"
+				;;
+			"-l")
+				echo "log_to_cerr = true"
+				command="${command} --config"
+				command="${command} node.logging.log_to_cerr=true"
+				;;
+			*)
+			 	command="${command} $i"
+				;;
+		esac
 	done
-elif [[ "${TEMP_OPTS[0]}" = 'bash' ]]; then
```

Comment:
was this intentionally removed without replacing with a sh alternative? see previous comment